backend/open_api_spec.yml

```diff
@@ -0,0 +1,790 @@
+        - tombstone
+        - assay
+        - organism
+        - tissue
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        name:
+          type: string
+        revision:
+          type: integer
+        collection_id:
+          $ref: "#/components/schemas/collection_uuid"
+        collection_visibility:
+          $ref: "#/components/schemas/visibility"
+        revised_at:
+          type: number
+        is_primary_data:
+          $ref: "#/components/schemas/is_primary_data"
+        X_normalization:
+          type: string
+        dataset_assets:
+          type: array
+          items:
+            $ref: "#/components/schemas/dataset_asset"
+        tombstone:
+          type: boolean
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        sex:
+          $ref: "#/components/schemas/ontology_element_array"
+        ethnicity:
+          $ref: "#/components/schemas/ontology_element_array"
+        development_stage:
+          $ref: "#/components/schemas/ontology_element_array"
+        dataset_deployments:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                type: string
+        linked_genesets:
+          type: array
+          items:
+            type: string
+        cell_type:
+          $ref: "#/components/schemas/ontology_element_array"
+        cell_count:
+          type: integer
+        X_approximate_distribution:
+          $ref: "#/components/schemas/distribution"
+        schema_version:
+          type: string
+    dataset_preview:
+      type: object
+      required:
+        - id
+        - curator_tag
+        - tissue
+        - assay
+        - disease
+        - organism
+        - tombstone
+      properties:
+        id:
+          $ref: "#/components/schemas/dataset_uuid"
+        curator_tag:
+          $ref: "#/components/schemas/curator_tag"
+        tissue:
+          $ref: "#/components/schemas/ontology_element_array"
+        assay:
+          $ref: "#/components/schemas/ontology_element_array"
+        disease:
+          $ref: "#/components/schemas/ontology_element_array"
+        organism:
+          $ref: "#/components/schemas/ontology_element_array"
+        tombstone:
+          type: boolean
+    dataset_asset:
+      type: object
+      properties:
+        id:
+          type: string
+        dataset_id:
+          type: string
+        filetype:
+          type: string
+          enum: [H5AD, RDS, CXG]
+        filename:
+          type: string
+        type:
+          type: string
+          enum: [REMIX, ORIGINAL]
+        s3_uri:
```

Comment:
How would a curator use the `s3_uri`?

Reply:
they don't really have a use for it.

Reply:
removed (`7b42f47d`)

Had left that in when I initially thought that those were already presigned urls. But since they aren't, I added in the `/curation/collections/{collection_uuid}/asset` route yesterday.